ArrayDemo.cls

```diff
@@ -0,0 +1,12 @@
+/*2. Method which will accept integer parameter and build an print array of upto that integer parameter.*/
+
+public class ArrayDemo {
+	public static void printArray(Integer arraysize){
+		List<Integer> a= new List<Integer>(); 
```

Comment:
@pranalimm Is it array of provided integer size? `If your accepts (integer) 5 as parameter then build array of any
datatype but array size should be accepted parameter so in this case your array size
should be 5 )`.

This is dynamic array which is serving any number of size.
The array is not limited to fix size. for example if I took size as 5 then I should not able to add element at location 6 but here in your code I am able to add as many as I can.